scout/parse/variant/coordinates.py

```diff
@@ -170,13 +170,20 @@ def parse_coordinates(variant, category, build="37"):
             end_chrom=end_chrom,
             svlen=variant.INFO.get("SVLEN"),
         )
+    elif category == "mei":
```

Comment:
This code and the following else condition are about the same. Can it be simplified?